frontend/src/Components/NAVBAR.js

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import "../navbar.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCamera } from "@fortawesome/free-solid-svg-icons";
+
+import Popup from "reactjs-popup";
+
+import { Grid, Row, Col, Navbar } from "react-bootstrap";
```

Comment:
to avoid name clash, import bootstrap navbar with alias, in this case eg. `Navbar as BSNavbar`